README.md

```diff
@@ -65,21 +66,8 @@ impl Pos {
 
 static GOAL: Pos = Pos(4, 6);
 let result = bfs(&Pos(1, 1), |p| p.neighbours(), |p| *p == GOAL);
-assert_eq!(result.expect("no path found").1, 4);
-```
-
-The second version does not declare a `Pos` type, makes use of more closures,
-and is thus shorter.
-
-``` rust
-use pathfinding::bfs;
 
-static GOAL: (i32, i32) = (4, 6);
-let result = bfs(&(1, 1),
-                 |&(x, y)| vec![(x+1,y+2), (x+1,y-2), (x-1,y+2), (x-1,y-2),
-                                (x+2,y+1), (x+2,y-1), (x-2,y+1), (x-2,y-1)],
-                 |&p| p == GOAL);
-assert_eq!(result.expect("no path found").1, 4);
+assert_eq!(result.expect("no path found"), 5);
 ```
```

Comment:
Why remove this example?

Reply:
I don't think this example is usefull because examples are just here to display how something work not how to optimize the number of lines of code you wrote